dask/dataframe/io/parquet/utils.py

```diff
@@ -431,3 +431,17 @@ def _scrub(i, p):
         "/".join(basepath),
         out_list,
     )  # use '/'.join() instead of _join_path to be consistent with split('/')
+
+
+def _flatten_seq(seq):
+    """ Simple utility to flatten nested list/tuple
+    """
+    t = []
+    if seq is None:
+        return t
+    for i in seq:
+        if not isinstance(i, (list, tuple)):
+            t.append(i)
+        else:
+            t.extend(_flatten_seq(i))
+    return t
```

Comment:
Maybe `dask.core.flatten` ?

Reply:
Nie - Thanks for pointing that out.  Using `dask.core.flatten`, what is the right way to handle `filters=[[('x', '>', 1), ('x', '<', 6)], [('x', '>', 3), ('x', '<', 8)]]`.  Perhaps something like `set(flatten(tuple(flatten(filters, container=list)), container=tuple))` ?

Reply:
I'm not sure.  It's been a while.  Also, no obligation here to use the function.  I was just cruising through the PR and it looked familiar.